setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Just looked at PyWavelets' [most recent build log]( https://readthedocs.org/projects/pywavelets/builds/5129158/ ) and it seems that it is actually building the library as there is a `python setup.py install --force` step that occurs, which I don't have in my WIP branch for pyFFTW. Is there anything in PyWavelets beyond using a `requirements.txt` file that is causing this?